### DISCUSSION TITLE
Issue with Custom Guard and Provider in Filament - "Route [login] not defined" Error

### DISCUSSION THREAD

**Discussion by jmendozaf** [MarJose123/filament-lockscreen#46] [Q&A]
In my application, I have set up a custom guard and provider (admin) in config/auth.php to handle authentication instead of the default web, primarily for Filament only. I thought this might be causing some conflicts with the lockscreen package, particularly with route definitions.

The specific error I am facing is: "Route [login] not defined." This error occurs when trying to lock my screen, and I suspect it is related to my custom admin guard and provider setup.

**Comment by MarJose123** [comment]
Can you check, if your default panel used the `web` guard if the plugin is working fine, and the same with the custom guard?

Can you share some error logs details for me to debug. 

The error you encountered is more on middleware of laravel `Authenticate`.


The lockscreen are using session it means that the user is not currently login.

**Comment by jmendozaf** [comment]
The default panel with the web guard and the plugin works fine! Also, the default or custom panel with the custom guard functions properly. The user is logged in. I think that the issue lies with the middleware using the plugin. Besides the issue with the lock screen, everything works as expected. I'm attaching some screenshots.
<img width="272" alt="Screenshot 2023-12-10 at 22 11 21" src="https://github.com/MarJose123/filament-lockscreen/assets/1133436/71d2b4a2-8ca6-47ae-a6cd-7c45ae52d3d3">
<img width="1366" alt="Screenshot 2023-12-10 at 22 11 59" src="https://github.com/MarJose123/filament-lockscreen/assets/1133436/314ce13c-9383-4771-90e0-91c656c86a07">
<img width="1339" alt="Screenshot 2023-12-10 at 22 13 34" src="https://github.com/MarJose123/filament-lockscreen/assets/1133436/88c6b8cb-8c77-43a7-aa5b-af0766060c40">

**Comment by MarJose123** [comment]
I see, I got it.

This is an issue I think within the validation and the plugin is not using the defined guard for authentication.

I will fix this during my free time, you are welcome also to PR.
